mantidproject/mantid
# Trivially Citable Citations: Base Class

**Description of work.**
The aim was to implement the base class of this design documentation:
https://github.com/mantidproject/documents/pull/67



**To test:**
Check that this works in the Script windows of MantidPlot and Workbench:
```python
from mantid.api import Citation 

cite = Citation("doi", "bibtex", "endnote", "url", "description")
print(cite.doi())
print(cite.bibtex())
print(cite.endnote())
print(cite.url())
print(cite.description())
```


Fixes #26160 . 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
